### PR TITLE
fix(isolation-dates-validation)

### PR DESCRIPTION
**Pull request by Noam6421**
Fix to bug: [1384](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1384/)

### CODE REVIEW

**Review by y-golde — DISMISSED** [review]

File: client/src/Utils/ClinicalDetails/symptomsUtils.ts [hunk 1]
```diff
@@ -26,4 +27,7 @@ export const getDatesToInvestigate = (doesHaveSymptoms: boolean, symptomsStartDa
     return [];
 }
 
-export const getMinimalSymptomsStartDate = (validationDate: Date) => subDays(new Date(validationDate), maxInvestigatedDays)
+export const getMinimalSymptomsStartDate = (validationDate: Date) => subDays(new Date(validationDate), maxInvestigatedDays);
+
```
Comment (nevobonneh) [inline client/src/Utils/ClinicalDetails/symptomsUtils.ts:31]: delete row

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts [hunk 2]
```diff
@@ -2,24 +2,27 @@ import * as yup from 'yup';
...
+import { getMinimalSymptomsStartDate, getMinimalStartIsolationDate, maxInvestigatedDays, maxIsolationDays } from 'Utils/ClinicalDetails/symptomsUtils';
 
 const requiredText = 'שדה זה הוא חובה';
 const StartDateAfterEndDateText = 'תאריך ההתחלה צריך להיות מוקדם יותר מתאריך הסיום';
 const EndDateBeforeStartDateText = 'תאריך הסיום צריך להיות מאוחר יותר מתאריך ההתחלה';
 const futureDateText = 'לא ניתן להכניס תאריך עתידי';
 const endDateBeforeValidationDateText = 'לא ניתן להכניס תאריך אחרי תאריך תחילת מחלה';
-const symptomsStartDateIsTooEarlyText = `לא ניתן להכניס תאריך יותר מ${maxInvestigatedDays} ימים לפני תאריך תחילת מחלה`
+const symptomsStartDateIsTooEarlyText = `לא ניתן להכניס תאריך יותר מ${maxInvestigatedDays} ימים לפני תאריך תחילת מחלה`;
+const isolationStartDateIsTooEarlyText = `לא ניתן להכניס תאריך יותר מ${maxIsolationDays} ימים לפני תאריך תחילת מחלה`
```
Comment (nevobonneh) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts:13]: ;

**Review by nevobonneh — CHANGES_REQUESTED** [review]

File: client/src/Utils/ClinicalDetails/symptomsUtils.ts (hunk 1 above)
Comment (Noam6421) [reply client/src/Utils/ClinicalDetails/symptomsUtils.ts:31]: 👍🏼

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts (hunk 2 above)
Comment (Noam6421) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts]: 👍🏼

**Review by nevobonneh — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**